src/citrine/resources/predictor.py

```diff
@@ -62,3 +63,33 @@ def check_for_update(self, predictor_id: UUID) -> Optional[Predictor]:
             return built
         else:
             return None
+
+    def auto_configure(self, training_data: DataSource, pattern="PLAIN") -> Predictor:
+        """[ALPHA] Create a default predictor for some training data.
+
+        This method will return an unregistered predictor generated by inspecting the
+        training data and attempting to automatically configure the predictor.
+        The automatic configuration tries to be as inclusive as possible.
+
+        Parameters
+        ----------
+        training_data: DataSource
+            The data to configure the predictor to model.
+        pattern: str
+            The predictor pattern to use, either "PLAIN" or "FORMULATION"
+
+        Returns
+        -------
+        Predictor
+            Automatically configured predictor for the training data
+
+        """
+        if pattern not in {"FORMULATION", "PLAIN"}:
+            msg = "Expected pattern: {}.  Expected 'FORMULATION' or 'PLAIN'.".format(pattern)
```

Comment:
Did you mean for this message to start with `Expected pattern` or is this a typo?

Reply:
Typo